llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -2625,6 +2645,8 @@ bool SPIRVInstructionSelector::selectIntrinsic(Register ResVReg,
     return selectFmix(ResVReg, ResType, I);
   case Intrinsic::spv_length:
     return selectLength(ResVReg, ResType, I);
+  case Intrinsic::spv_degrees:
+    return selectDegrees(ResVReg, ResType, I);
```

Comment:
Assuming it does take 1 operand, can you use selectExtInst here instead of a custom function?

Reply:
Yep, you are right and I now realize I should also support the option to use the opencl lowering path which can be done using `selectExtInst`.